locales/en.json

```diff
@@ -25,19 +25,23 @@
   "https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/lines-330-331-eligible-medical-expenses-you-claim-on-your-tax-return.html#mdcl_xpns": "https://www.canada.ca/en/revenue-agency/services/tax/individuals/topics/about-your-tax-return/tax-return/completing-a-tax-return/deductions-credits-expenses/lines-330-331-eligible-medical-expenses-you-claim-on-your-tax-return.html#mdcl_xpns",
   "https://www.canada.ca/en/revenue-agency/services/tax/individuals/community-volunteer-income-tax-program.html": "https://www.canada.ca/en/revenue-agency/services/tax/individuals/community-volunteer-income-tax-program.html",
   "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/netfile-overview/certified-software-netfile-program.html": "https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/netfile-overview/certified-software-netfile-program.html",
-  "This service is to claim tax benefits online.": "This service is to claim tax benefits online.",
-  "Claim Tax Benefits": "Claim Tax Benefits",
+  "service.name": "Claim Tax Benefits",
+  "start.banner": "Keep your notification letter with you. You need information on that letter to use this service.",
+  "start.p1.1": "Welcome to the Canada Revenue Agency’s new",
+  "start.p1.2": "service",
+  "start.p2.1": "This service is to claim tax benefits online.",
+  "start.p2.2": "You are eligible for this service if:",
+  "start.li1": "You earned under $35,000 in 2018 (from employment, pension, RRSP withdrawals, or from benefits and credits you received)",
+  "start.li2": "You do not have self-employment income to report",
+  "start.li3": "You do not earn investment income (except for nominal interest income from Canadian sources)",
+  "start.p3": "In order to claim tax benefits online, you must have the following:",
```

Comment:
even though it's just the one paragraph, do we want to just keep it consistent with `start.p3.1`?